### PR TITLE
Frontispiece v4.1 updates

### PR DESCRIPTION
**Pull request by kingthorin**
Per: OWASP/wstg#249

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: document/1_Frontispiece/1_Frontispiece.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
-# OWASP Web Security Testing Guide 5.0
+# OWASP Web Security Testing Guide 4.1
 
 ## Welcome
 
-Placeholder-TBC
+“Open and collaborative knowledge: that is the OWASP way.”
```
Comment (kingthorin) [inline document/1_Frontispiece/1_Frontispiece.md]: Shall I make this a MD quote?

Comment (ThunderSon) [reply document/1_Frontispiece/1_Frontispiece.md]: Yes please

Comment (kingthorin) [reply document/1_Frontispiece/1_Frontispiece.md]: Done

Comment (victoriadrake) [reply document/1_Frontispiece/1_Frontispiece.md]: If it's a quote, who said it?

File: document/1_Frontispiece/1_Frontispiece.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+# OWASP Web Security Testing Guide 4.1
 
 ## Welcome
 
-Placeholder-TBC
+> Open and collaborative knowledge: that is the OWASP way.
 
-## Version 5.0
+With V4 we realized a new guide that will be the standard de-facto guide to perform Web Application Penetration Testing. \
+-- Matteo Meucci
```
Comment (victoriadrake) [inline document/1_Frontispiece/1_Frontispiece.md]: This is a quote, it seems.

File: document/1_Frontispiece/1_Frontispiece.md [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 
-Placeholder-TBC
+OWASP thanks the many authors, reviewers, and editors for their hard work in bringing this guide to where it is today. If you have any comments or suggestions on the Testing Guide, please feel free to open an Issue or submit a fix/contribution via Pull Request to our [GitHub repo](https://github.com/OWASP/wstg/).
+
+## Version 4.1
+
+This minor release represents a transitional step between the 2014 release of v4 via the OWASP wiki, and the preparation of v5 which is being assembled via GitHub.
```
Comment (victoriadrake) [inline document/1_Frontispiece/1_Frontispiece.md]: "which is being assembled" isn't quite right. There's more than just assembly going on. Maybe "...the preparation of v5, currently underway on GitHub."?

Comment (kingthorin) [reply document/1_Frontispiece/1_Frontispiece.md]: Sure, that's fair. I waffled over that a bunch, I think I re-wrote it about 3 times then just decided to stick with something and get feedback :+1:

File: document/1_Frontispiece/1_Frontispiece.md (hunk 2 above)
Comment (kingthorin) [reply document/1_Frontispiece/1_Frontispiece.md]: Arg it got split on multiple lines.....tks

Comment (victoriadrake) [reply document/1_Frontispiece/1_Frontispiece.md]: Oh that makes more sense.

**Comment by kingthorin** [comment]
Thanks @victoriadrake

**Pull request merged**